<?xml version="1.0" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="6.935686349868774">
	<testsuite disabled="0" errors="0" failures="1" name="meraki_mx_l2_interface-f5n5253z" skipped="0" tests="2" time="6.935686349868774">
		<testcase classname="/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_mx_l2_interface-2pec7ti6-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_mx_l2_interface/tasks/main.yml:7" name="[localhost] meraki: meraki_mx_l2_interface : Create network with type appliance auth_key={{ auth_key }}, state=present, org_name={{test_org_name}}, net_name={{test_appliance_net_name}}, type=appliance, timezone=America/Chicago" time="6.842255">
			<system-out>{
&quot;changed&quot;: true,
&quot;data&quot;: {
&quot;enrollment_string&quot;: null,
&quot;id&quot;: &quot;N_624874448297687172&quot;,
&quot;name&quot;: &quot;Home - Appliance&quot;,
&quot;notes&quot;: null,
&quot;organization_id&quot;: &quot;133277&quot;,
&quot;product_types&quot;: [
&quot;appliance&quot;
],
&quot;tags&quot;: [],
&quot;time_zone&quot;: &quot;America/Chicago&quot;,
&quot;url&quot;: &quot;https://n110.meraki.com/Home-Appliance/n/ac96vdUb/manage/usage/list&quot;
},
&quot;deprecations&quot;: [
{
&quot;msg&quot;: &quot;Distribution Ubuntu 18.04 on host localhost should use /usr/bin/python3, but is using /usr/bin/python for backward compatibility with prior Ansible releases. A future Ansible release will default to using the discovered platform python for this host. See https://docs.ansible.com/ansible/2.10/reference_appendices/interpreter_discovery.html for more information&quot;,
&quot;version&quot;: &quot;2.12&quot;
}
],
&quot;response&quot;: &quot;OK (unknown bytes)&quot;,
&quot;status&quot;: 201
}</system-out>
		</testcase>
		<testcase classname="/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_mx_l2_interface-2pec7ti6-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_mx_l2_interface/tasks/main.yml:18" name="[localhost] meraki: meraki_mx_l2_interface : Claim a device into a network auth_key={{auth_key}}, org_name={{test_org_name}}, net_name={{test_appliance_net_name}}, serial={{serial_appliance}}, state=present" time="0.093431">
			<failure message="The task includes an option with an undefined variable. The error was: 'serial_appliance' is undefined

The error appears to be in '/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_mx_l2_interface-2pec7ti6-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_mx_l2_interface/tasks/main.yml': line 18, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - name: Claim a device into a network
    ^ here
" type="failure">{
&quot;msg&quot;: &quot;The task includes an option with an undefined variable. The error was: 'serial_appliance' is undefined\n\nThe error appears to be in '/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_mx_l2_interface-2pec7ti6-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_mx_l2_interface/tasks/main.yml': line 18, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n  - name: Claim a device into a network\n    ^ here\n&quot;
}</failure>
		</testcase>
	</testsuite>
</testsuites>
